modules/nw-anp-banp-metrics.adoc

```diff
@@ -0,0 +1,181 @@
+// Module included in the following assemblies:
+//
+// * list of assemblies where this module is included:
+// networking/openshift_network_security/adminnetworkpolicy
+
+:_mod-docs-content-type: CONCEPT
+[id="anp-banp-metrics_{context}"]
+= AdminNetworkPolicy and BaselineAdminNetworkPolicy metrics
+
+The following table describes 12 metrics that can support monitoring your ANP and BANP.
+
+[cols="1,1,1"]
+|===
+| Name |labels/action |Explanation
+
+|`ovnkube_controller_admin_network_policies`
+|Not applicable
+|The total number of `AdminNetworkPolicy` in the cluster.
+
+|`ovnkube_controller_baseline_admin_network_policies`
+|Not applicable
+|The total number of `BaselineAdminNetworkPolicy` in the cluster. Values should be 0 or 1.
+
+|`ovnkube_controller_admin_network_policies_rules`
+|*`direction`: specifies either ingress or egress.
+* `action`: specifies either `pass`, `allow`, or `deny`.
+|The total number of rules across all `AdminNetworkPolicy` in the cluster
+
+|`ovnkube_controller_baseline_admin_network_policies_rules`
+|*`direction`: specifies either ingress or egress.
+* `action`: specifies either `pass or `deny`.
```

Comment:
Same as above, it is kind of reinforcement for the users to always capitalize the actions.